src/pages/EmployeeCvsPage/EmployeeCvsPage.tsx

```diff
@@ -0,0 +1,7 @@
+import EmployeeTabs from '../../components/EmployeeTabs';
```

Comment:
What the idea of this component? For me it is looks like you just reexport EmployeeTabs component

Reply:
It's ok, these pages are just not ready yet